api/func-e.go

```diff
@@ -61,6 +60,27 @@ func Out(out io.Writer) RunOption {
 	}
 }
 
+// EnvoyOut sets the writer for Envoy stdout
+func EnvoyOut(w io.Writer) RunOption {
```

Comment:
api changes here.. only 2 of them, and they are additional. In general, this package is a bomb as it ties api to impl, but we can shave that yak in a later PR.